exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileResources.java

```diff
@@ -105,6 +109,25 @@ public String getTime() {
       return format.format(time);
     }
 
+    public long getStartTime() {
+      return startTime;
+    }
+
+    public long getEndTime() {
+      return endTime;
+    }
+
+    public String getDuration() {
```

Comment:
That could be one way to work around it. It would be good to have the human-readable formatting done through only one static method. I'm still trying to familiarize myself with the way these classes and the freemarker templates interact, so let me see if I can do with a wrapper. :)